src/auditor/compiler_abi.jl

```diff
@@ -51,7 +51,8 @@ function check_libgfortran_version(oh::ObjectHandle, platform::Platform; verbose
 end
 
 """
-    detect_libstdcxx_version(oh::ObjectHandle, platform::Platform)
+    detect_libstd_ver
```

Comment:
Extra new line

Reply:
That will teach me for using the GitHub web interface...